poedit.html

```diff
@@ -210,6 +212,17 @@ <h1>Contact</h1>
 <script type='text/javascript'>
 getRGB();
 PoEdit.init();
+
+function limitInput(limit) {
+    if(this.value > limit) {
+        this.value = limit;
+    }
+}
+
+function isValidNumberEntry() {
+    // keyCode 8: Backspace; keyCode 46: Delete
+    return (event.keyCode === 8 || event.keyCode === 46) ? true : !isNaN(Number(event.key));
+}
```

Comment:
I think this won't work if the user pastes a number into the input field. Wouldn't it be better to just check if the value of the element is a number, i.e. something along the lines of
`!isNaN(document.getElementById("area-level-input").value)`
Not a perfect solution either since "1e2" would be a valid number then.

Reply:
Could make it 'onpaste="return false;"' I suppose to prevent the pasting problem.

What's the !isNaN(document.getElementById("area-level-input").value) for? Could you specify which part it replaces? 

Also odds are it won't be merged or anything - I think the owner of the repository isn't looking at it :/

Reply:
Hey there, sorry for the really late reply. I'm totally swamped with real world responsibilities at the moment and don't have time to maintain this project. As for your PR, I see two problems with it: 

a) I try not to add restrictions on what is currently possible in the game. For example, I consciously didn't add a dropdown for item class. This is to ensure that whenever something is added to the game, people can still use the tool and don't have to wait for an update. Only changes to the actual filter rules should require editor updates. So I don't think you should limit the area level to 100.

b) You are blocking any non-digit keys other than backspace and delete. What about arrow keys, what about shift for selecting, what about Ctrl+A? Ctrl-C? Ctrl-V? Will this restriction still work with right-click / paste? I don't think we should try to mess around with the input handling and instead validate the result. So your limitInput function should check if it's a number and if not, revert to the last valid input.